x-pack/plugins/timelines/common/utils/field_formatters.ts

```diff
@@ -42,7 +43,8 @@ export const isGeoField = (field: string) =>
   field.includes('geo.location') || field.includes('geoip.location');
 
 export const isRuleParametersFieldOrSubfield = (field: string, prependField?: string) =>
-  prependField?.includes(ALERT_RULE_PARAMETERS) || field === ALERT_RULE_PARAMETERS;
+  (prependField?.includes(ALERT_RULE_PARAMETERS) || field === ALERT_RULE_PARAMETERS) &&
+  !nonFlattenedFormatParamsFields.includes(field);
```

Comment:
nit: Is `isRuleParametersFieldOrSubfield` called in a hot path? If so, it might make sense to switch from array checks here to object checks e.g.:

```js
const nonFlattenedFormatParamsFields = {
  related_integrations: true,
  threat_mapping: true
};

// then for the check
if (!nonFlattenedFormatParamsFields[field]) { ... }
```